README.md

```diff
@@ -1,2 +1,92 @@
-# module-ballerina-observe
-Ballerina observability components exposed for users
+
+Ballerina Observe Internal Library
+===================
+
+[![Build](https://github.com/ballerina-platform/module-ballerina-http/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/build-timestamped-master.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerina-http/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-http.svg)](https://github.com/ballerina-platform/module-ballerinai-observe/commits/master)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-observe/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-observe)
+
+lerina project and enable observability features.
```

Comment:
There seems to be something wrong with this sentence 🤔

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-observe/pull/36/commits/f1ac7a646e1fbb9c4d5546f7755da6069af062ea